native/packages/react-native-bpk-component-phone-input/src/BpkDialingCodeAccessoryView.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import React, { type Node } from 'react';
+import {
+  type Element,
+  Image,
+  Platform,
+  StyleSheet,
+  TouchableNativeFeedback,
+  View,
+} from 'react-native';
+
+import {
+  colorGray100,
+  borderSizeSm,
+  spacingSm,
+  spacingBase,
+} from 'bpk-tokens/tokens/base.react.native';
+import BpkText from 'react-native-bpk-component-text';
+import BpkIcon, { icons } from 'react-native-bpk-component-icon';
+import BpkTouchableOverlay from 'react-native-bpk-component-touchable-overlay';
+import { type Code } from './common-types';
+import BpkFlag from './BpkFlag';
+
+const styles = StyleSheet.create({
+  accessoryView: {
+    flex: 1,
+    flexDirection: 'row',
+    borderColor: colorGray100,
+    borderBottomWidth: borderSizeSm,
+    alignItems: 'center',
+  },
+  outerAccessoryView: {
+    justifyContent: 'flex-end',
+  },
+  flag: {
+    marginEnd: spacingSm,
+  },
+  dialingCode: {
+    marginEnd: spacingSm,
+  },
+  nonEditable: {
+    color: colorGray100,
+  },
+});
+
+type FeedbackProps = {
+  children: Node,
+  onPress: () => void,
```

Comment:
shouldn't the return type for this be mixed so that folks can do inline implicit return style functions? i.e. `onPress={() => this.doSomething(locallyScopedValue)}`

Reply:
Yeah good shout